Makefile

```
@@ -617,3 +618,65 @@ clean-asl-pseudocode:
 
 asldoc: $(BENTO)
 	@ $(MAKE) $(MFLAGS) -C $(@D)/asllib/doc all
+
+$(V).SILENT:
+
+litmus-test:: litmus-cata-aarch64-test-std
+litmus-cata-aarch64-test-std: TEMP_DIR:=$(shell mktemp -d)
+litmus-cata-aarch64-test-std:
+	$(LITMUS) -mode std -a 4 catalogue/aarch64/tests/@all -o $(TEMP_DIR)
+	make -C $(TEMP_DIR) -j $(J)
+	$(RM) -r $(TEMP_DIR)
+	@ echo "litmus7 in -mode std catalogue aarch64 tests: OK"
+
+litmus-test:: litmus-cata-aarch64-test-presi
+litmus-cata-aarch64-test-presi: TEMP_DIR:=$(shell mktemp -d)
+litmus-cata-aarch64-test-presi:
+	$(LITMUS) -mode presi -a 4 catalogue/aarch64/tests/@all -o $(TEMP_DIR)
+	make -C $(TEMP_DIR) -j $(J)
+	$(RM) -r $(TEMP_DIR)
+	@ echo "litmus7 in -mode presi catalogue aarch64 tests: OK"
+
+ifeq ($(OS),Darwin)
+KUT_CONFIG_PARAMS=--page-size=4k --cross-prefix=aarch64-elf-
+endif
+KUT_DIR:=$(shell mktemp -d)
+
+litmus-dep:
+	cd $(KUT_DIR); \
+	git clone -q git@gitlab.com:kvm-unit-tests/kvm-unit-tests.git; \
```

Access to the kvm-unit-test repository looks easier using http. Namely, public key access does not work for me, probably by lack of some configuration:
```suggestion
	git clone -q https://gitlab.com/kvm-unit-tests/kvm-unit-tests.git; \
```